src/ReservoirPair.sol

```diff
@@ -336,7 +339,6 @@ abstract contract ReservoirPair is ReservoirERC20 {
         returns (uint104 rReserve0, uint104 rReserve1)
     {
         if (address(assetManager) == address(0)) {
-            // PERF: Is assigning to rReserve0 cheaper?
```

Comment:
attempted assigning to return value and introducing an else for the rest of the procedure. 

There was no difference at all so this is prolly optimized by the compiler + optimizer already